cmd/libs/go2idl/conversion-gen/generators/conversion.go

```diff
@@ -0,0 +1,869 @@
+/*
```

Comment:
@ncdc - would love your analysis of https://github.com/kubernetes-incubator/service-catalog/issues/223